docs/src/intro_to_catalyst/using_catalyst.md

```diff
@@ -110,7 +110,7 @@ conditions. We can build such mappings in several ways. One is to use Julia
 ```@example tut1
 pmap  = (:α => .5, :K => 40, :n => 2, :δ => log(2)/120,
          :γ => 5e-3, :β => log(2)/6, :μ => log(2)/60)
-u₀map = [:m₁ => 0., :m₂ => 0., :m₃ => 0., :P₁ => 20., :P₂ => 0., :P₃ => 0.]
+u₀map = [:m₁ => 0., :m₂ => 0., :m₃ => 0., :P₁ => 20., :P₂ => 0., :P₃ => 0.];
```

Comment:
```suggestion
u₀map = [:m₁ => 0., :m₂ => 0., :m₃ => 0., :P₁ => 20., :P₂ => 0., :P₃ => 0.]
nothing   # hide
```
Instead of adding a `;` which will appear in the docs, I think you can do the above. Give it a try locally and check if that effectively suppresses the output.